lib/bundler/settings.rb

```diff
@@ -129,6 +129,14 @@ def with
       get_array(:with)
     end
 
+    def plugins
+      @global_config[key_for(:plugins)] ? @global_config[key_for(:plugins)].split(":") : []
```

Comment:
Global settings are stored in ~/.bundle/config 

We wouldn't want to lose information about user level plugins.